src/Plugin.php

```diff
@@ -92,6 +92,14 @@ public static function init() {
     if (static::isPluginActive('bj-lazy-load/bj-lazy-load.php')) {
       add_filter('bjll/skip_classes', __CLASS__ . '::bjll_skip_classes');
     }
+
+    // Adds wp-graphql support for product variation additional gallery images.
+    if (static::isPluginActive('wp-graphql/wp-graphql.php')
+    && static::isPluginActive('wp-graphql-woocommerce/wp-graphql-woocommerce.php')
```

Comment:
@sun 
Do we need these checks? 

If this filter is not declared `graphql_register_types` the code will not execute, making the plugin check a bit redundant.

Reply:
Yeah, if all the integration logic can be put inside of the `graphql_register_*` hook callbacks, then of course that is ideal and no further checks for plugins are necessary, because the hook won't be invoked by something else (or if it will, then it'll just work magically).

Seems to be the case here so we can remove the condition checking for the plugin.